runtime/vm/JVMImage.hpp

```diff
@@ -76,18 +50,51 @@ public
 	char *_dumpFileName;
 	bool _isWarmRun;
 
-	bool initializeMonitor();
+	bool initializeMonitor(void);
 
 	void* allocateImageMemory(UDATA size);
 	void* reallocateImageMemory(UDATA size);
-	void* initializeHeap();
+	void* initializeHeap(void);
 
-	bool allocateImageTableHeaders();
+	bool allocateImageTableHeaders(void);
 	void* allocateTable(ImageTableHeader *table, uintptr_t tableSize);
 	void* reallocateTable(ImageTableHeader *table, uintptr_t tableSize);
 
-	bool readImageFromFile();
-	bool writeImageToFile();
+	bool readImageFromFile(void);
+	bool writeImageToFile(void);
+protected:
+public:
+	static const UDATA INITIAL_HEAP_SIZE;
+	static const UDATA INITIAL_IMAGE_SIZE;
+	
+	/*
+	 * Function Members
+	 */
+private:
+protected:
+	void *operator new(size_t size, void *memoryPointer) { return memoryPointer; }
+public:
+	JVMImage(J9JavaVM *vm);
+	~JVMImage(void);
```

Comment:
My only problem is with the destructor having a void param. I haven't seen it in the codebase anywhere and I couldn't find specifics on coding standards for constructor/destructor